webmunk-ext/src/background/worker.js

```diff
@@ -6,10 +6,63 @@ import { wmSessionMgr } from '@webmunk/utils';
 import "@webmunk/extension-ads/worker.js";
 import "@webmunk/cookies-scrapper-module/worker";
 
-const appMgr =  {
-  initialize:function(){
-    messenger?.addReceiver('appMgr', this);
-  }
+const appMgr = {
+  surveys: [],
+  completedSurveys: [],
+
+  initialize: async function () {
+    messenger?.addReceiver("appMgr", this);
+
+    chrome.storage.local.get('completedSurveys', (result) => {
```

Comment:
Сreate method initSurveys and move chrome.storage.local.get there.
Also chrome.storage.local.get return promise, so you can use async/await instead of callbacks.